alpenhorn_chime/__init__.py

```diff
@@ -0,0 +1,23 @@
+"""Alpenhorn extensions for CHIME."""
+
+import peewee as pw
+
+from .inst import set_inst, ArchiveInst
+
+
+def register_extension() -> dict:
+    """Return alpenhorn extension data.
+
+    This module extends ArchiveAcq to add the `inst` field.
+    """
+    return {
+        "model": [
+            {
+                "table": "ArchiveAcq",
+                "setter": set_inst,
+                "fields": {
+                    "inst": pw.ForeignKeyField(ArchiveInst, backref="acqs", null=True)
+                },
+            }
+        ]
+    }
```

Comment:
I thought we had decided (several times) to move `ArchiveInst` from being keyed on `ArchiveAcq` to being on the various `XAcqInfo` tables. Is that not happening any more?

This was one of the bigger DB schema changes we need to do, but I think it's reasonable and probably better than hacking an extra field into the core alpernhorn tables.

Reply:
Yes, indeed we have and I say this here doesn't change that.

We talked a bit about this in person, but to summarise, removing `ArchvieInst` from alpenhorn is a good idea, and this doesn't change that.  Really, this is just a way for CHIME to shove that bit of information into the `ArchiveInst` table where it's convenient to store it.

Alpenhorn doesn't care about the column at all: this extension is only used when importing new files.  It's entirely analagously to how it doesn't really care about info tables either.  Everywhere else in alpenhron, this extension is ignored.

Even within the CHIME context here, the column allows NULL, meaning acquisitions are permitted to not have an instrument, if they don't want it.